test/contracts/exec-dest-ctx-recursive-parent/exec-dest-ctx-recursive-parent.arwen

```diff
@@ -0,0 +1 @@
f60027f7f017f6000017f60017f017e60087e7f7f7f7f7f7f7f017f60000002ef010b03656e760c73746f7261676553746f7265000003656e760666696e697368000103656e760e626967496e74536574496e743634000203656e7609626967496e74416464000303656e760b7369676e616c4572726f72000103656e761573746f7261676547657456616c75654c656e677468000403656e760b73746f726167654c6f6164000503656e760f6765744e756d417267756d656e7473000603656e7610696e743634676574417267756d656e74000703656e7614657865637574654f6e53616d65436f6e74657874000803656e761a626967496e7453746f7261676553746f7265556e7369676e65640005030201090405017001010105030100020608017f0141c08a040b071d02066d656d6f7279020010706172656e7443616c6c734368696c64000b0acb0601c80603027f017e067f23808080800041c0006b22002480808080004101210102401087808080004101460d00200041286a41002f01e8888080003b0100200041206a41002903e088808000370300200041002903d888808000370318200041002903d088808000370310200041106a41191084808080000b41001088808080002102200041306a41002d00a0888080003a0000200041286a410029039888808000370300200041206a220341002903908880800037030020004100290388888080003703182000410029038088808000370310200041086a220441002f00a9888080003b010020042002a72205200541ff01712206410a6e2207410a6c6b41307222083a0000200041002900a1888080003703002000200641e4006e41307222043a0014200020043a000620002007410a7041307222063a0015200020063a0007200020083a0016200041d0003a0000200041d0003a0010200041106a200041091080808080001a200041002800b288808000360017200041002900ab88808000370310200020083a0019200020063a0018200020043a0017200041d0003a0010200041106a410a108180808000024041908a808000108580808000450d0041908a808000200041106a1086808080001a20002d001041016a21010b200020013a001041908a808000200041106a41011080808080001a41002802e4898080002108412a420110828080800020082008412a108380808000200341002d0080898080003a0000200041002903f888808000370318200041002903f08880800037031002400240200242ff0183500d0041002005417f6a3a00b88a808000024041002903908980800041a08980800041f089808000200041106a4110410141b48a80800041b88a808000108980808000220841014b0d000240024020080e020001000b200041046a41002d00ba888080003a0000200041002800b688808000360200200041041081808080000c030b200041046a41002d00bf888080003a0000200041002800bb88808000360200200041041081808080000c020b200041046a41002d00c4888080003a0000200041002800c088808000360200200041041081808080000c010b41c08980800041002802e489808000108a808080001a0b200041c0006a2480808080000b0bbf0203004180080b8101586b65794e4e4e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e005876616c75654e4e4e005866696e6973684e4e4e0073756363006661696c00756e6b6e00000000000000000000000077726f6e67206e756d626572206f6620617267756d656e7473000000000000006368696c6443616c6c73506172656e7400004190090ba801a08601000000000000000000000000006368696c6453432e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e2e726563757273697665497465726174696f6e426967436f756e7465722e2e2e2e000000005800000000000000000000000000000000000000000000000000000000000000000000000000000000000005726563757273697665497465726174696f6e436f756e7465722e2e2e2e2e2e2e00000000010000000041b80a0b0100@0500
```

Comment:
Just a note, the new version of erdpy won't generate `*.hex.arwen` files anymore.